### PR TITLE
Do-not-distrub configurable signals

### PR DESCRIPTION
**Pull request by ricci**
I use the do-not-disturb module with `dunst`, and prefer to use `SIGUSR2` to bring it out of DND mode (`SIGTERM` actually causes the process to exit). This patch makes the set of signals configurable.

### CODE REVIEW

File: py3status/modules/do_not_disturb.py [hunk 1]
```diff
@@ -42,6 +46,8 @@ class Py3status:
     format = '{state}'
     notification_manager = 'dunst'
     refresh_interval = 0.25
+    signal_off = 'SIGTERM'
+    signal_on = 'SIGUSR1'
```
Comment (maximbaz) [inline py3status/modules/do_not_disturb.py]: Since these are dunst-specific, I wonder if the parameter names should have `dunst_` prefix.

**Comment by ricci** [comment]
For what it's worth: I'm not sure what version of `dunst` has this behaviour for `SIGTERM`, as the mainline `dunst` just registers this as an exit handler: https://github.com/dunst-project/dunst/blob/master/src/dunst.c#L189

However, I didn't want to change the default behaviour since presumably, it works as intended for some people

File: py3status/modules/do_not_disturb.py (hunk 1 above)
Comment (ricci) [reply py3status/modules/do_not_disturb.py]: Sure, this seems like a good idea, I'll change it.

**Comment by maximbaz** [comment]
I think such behavior works only when dunst is managed by its systemd service, which restarts dunst immediately after it gets killed 🙂 Thanks for not changing the default behavior.

**Comment by ricci** [comment]
Ah, I see, that makes sense, I'm not running `dunst` from `systemd`

File: py3status/modules/do_not_disturb.py [hunk 2]
```diff
@@ -5,6 +5,10 @@
 A left mouse click will toggle the state of this module.
 
 Configuration parameters:
+    dunst_signal_off: Signal to send to the dunst process to disable "Do Not Distrub"
+        (default 'SIGTERM')
+    dunst_signal_on: Signal to send to the dunst process to enable "Do Not Distrub"
+        (default 'SIGUSR1')
```
Comment (lasers) [inline py3status/modules/do_not_disturb.py]: s/Distrub/Disturb. And maybe a period at end too. /ocd

**Comment by tobes** [comment]
@ricci thanks for this new functionality

@lasers @maximbaz thanks as ever for your inputs

**Pull request merged**